components/icons/Envelope.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
@AndyClifford where is this icon used?

Reply:
Cheers - Its not used anymore as some of the initial design was removed